Cubical/Data/Nat/Divisibility.agda

```diff
@@ -0,0 +1,98 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Data.Nat.Divisibility where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Isomorphism
+
+open import Cubical.Data.Sigma
+open import Cubical.Data.Empty as ⊥
+
+open import Cubical.HITs.PropositionalTruncation as PropTrunc
+
+open import Cubical.Data.Nat.Base
+open import Cubical.Data.Nat.Properties
+open import Cubical.Data.Nat.Order
+
+private
+  variable
+    l m n : ℕ
+
+_∣_ : ℕ → ℕ → Type₀
+m ∣ n = ∥ Σ[ c ∈ ℕ ] c * m ≡ n ∥
```

Comment:
It would be nice to have a notation for this (\exists)

Reply:
I mean for the truncated Sigma

Reply:
There is the ∃[]-syntax in Foundations.Logic that could be use here

Reply:
The ∃[]-syntax in `Logic` assumes that the RHS of the sigma is a proposition, so I just defined a new version. I also incorporated it in a number of places in the library.

Reply:
Nice! We should add unique existence as well (iscontr Sigma), not sure if this should go with exists or in the Prelude